app/services/induction/change_preferred_email.rb

```diff
@@ -2,21 +2,8 @@
 
 class Induction::ChangePreferredEmail < BaseService
```

Comment:
Could be completely missing it but is this being used anywhere? If not, should it be in the update email?

Reply:
good spot, completely overlooked that